plone/app/portlets/portlets/navigation.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 from Acquisition import aq_inner, aq_base, aq_parent
 from ComputedAttribute import ComputedAttribute
+from plone import api
```

Comment:
I think here it is not ok. I hate this, but as long we do not have a clear definition what is consumer and what is provider for plone.api we can not use it.